test/NuGet.Core.Tests/NuGet.Packaging.Test/DependeciesGroupsForEachTFMRuleTests.cs

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Frameworks;
+using NuGet.Packaging.Rules;
+using Xunit;
+using NuGet.RuntimeModel;
+using NuGet.Client;
+using NuGet.ContentModel;
+using NuGet.Packaging.Core;
+
+namespace NuGet.Packaging.Test
+{
+    public class DependeciesGroupsForEachTFMRuleTests
+    {
+        ManagedCodeConventions _managedCodeConventions = new ManagedCodeConventions(new RuntimeGraph());
+        ContentItemCollection _collection = new ContentItemCollection();
```

Comment:
These two fields are not used anywhere

Reply:
They extract the TFMs. Scroll down

Reply:
they do in the validation class, but I don't see any usage in this test class.